templates/home/profile.html.twig

```diff
@@ -0,0 +1,26 @@
+{% extends 'base.html.twig' %}
+{% block body %}
+    <div class="container">
+        <h1 class="text-center my-5">Mon profil</h1>
+        <div class="row">
+            <div class="col">
+                <div class="card">
+                    <div class="card-header">
+                        Mes informations de connexion
+                    </div>
+                    <div class="card-body">
+                        <h5 class="card-title">Mon adresse email : {{ app.user.email }}</h5>
+                        <p class="card-text">Mon type d'accès :
+                            {% if app.user.roles.0 == 'ROLE_ADMIN' %}
+                                Administrateur
```

Comment:
pourquoi pas faire direct via la constante ROLES? tu peux acceder à une constante depuis un Twig (voir doc de Twig)

Reply:
je n'ai pas de constante ROLES car je n'ai qu'un ROLE_ADMIN mis en place dans le UserFixture, le ROLE_USER étant celui mis par défaut dans le getUser de l'entité User

Reply:
si dans ProfileController tu as une constante ! Mais en fait je te dis des bétises, le mieux c'est d'utiliser is_granted https://symfony.com/doc/current/reference/twig_reference.html#is-granted
`{% if is_granted('ROLE_XXX') %}`

Reply:
le problème de is_granted c'est que pour l'admin je vais avoir 2 rôles: rôle admin et rôle user comme tout le monde a le rôle user par défaut, or moi je ne veux que le rôle le plus haut je ne veux pas lister tous ses rôles

Reply:
non en fait je voyais plutot 
```
 {% if is_granted('ROLE_ADMIN') %} 
Administrateur
{% else if is_granted('ROLE_USER') %} 
Utilisateur 
{% endif %}
```

Reply:
si je fais ça, ça m'affiche Administrateur, Utilisateur pour mon admin